planning/scenario_planning/common/motion_velocity_optimizer/src/motion_velocity_optimizer.cpp

```diff
@@ -661,18 +708,77 @@ void MotionVelocityOptimizer::updateExternalVelocityLimit(const double dt)
   }
 }
 
-void MotionVelocityOptimizer::timerCallback(const ros::TimerEvent & e)
+void MotionVelocityOptimizer::timerCallback()
 {
-  const double dt = (e.current_expected - e.last_expected).toSec();
+  const double dt = 1.0 / external_velocity_limit_update_rate_;
```

Comment:
was there some check that `external_velocity_limit_update_rate_` > 0?

Reply:
add a check when receiving the parameter.